src/components/Footer/Footer.tsx

```diff
@@ -56,9 +60,11 @@ const Footer = () => {
                     </Link>
                 </li>
                 <li className="footer-link">
-                    <Link to="" className="footer-link-content">
-                        <NewsletterModal />
-                    </Link>
+                    <button
+                        className="footer-link-content"
+                        onClick={showNewsletterModal}>
+                        {t('footer.newsletter')}
+                    </button>
```

Comment:
Footer is not visible anymore (removed). I think I had to left some files since it was affecting the UI when removing

Reply:
indeed